AsyncDisplayKit/ASNetworkImageNode.mm

```diff
@@ -281,21 +281,31 @@ - (void)displayWillStart
 
 /* visibileStateDidChange in ASMultiplexImageNode has a very similar implementation. Changes here are likely necessary
  in ASMultiplexImageNode as well. */
-- (void)visibleStateDidChange:(BOOL)isVisible
+- (void)didEnterVisibleState
 {
-  [super visibleStateDidChange:isVisible];
-
+  [super didEnterVisibleState];
+  
   if (_downloaderFlags.downloaderImplementsSetPriority) {
     ASDN::MutexLocker l(__instanceLock__);
```

Comment:
If we access the `_downloaderFlags` do we not also need to hold the `__instanceLock__ `?. Also in `didExitVisibleState`